bin/twemproxy_exporter

```diff
@@ -5,6 +5,10 @@ require 'prometheus/client/rack/exporter'
 require 'rack'
 require 'yaml'
 
+lib = File.expand_path('../../lib', __FILE__)
+$LOAD_PATH.unshift(lib) if File.directory?(lib) && !$LOAD_PATH.include?(lib)
```

Comment:
Is there a reason (backwards compatibility?) to prefer this to `require_relative`?  I've seen it in several projects, but I'm still a bit nervous about mucking with `$LOAD_PATH`...